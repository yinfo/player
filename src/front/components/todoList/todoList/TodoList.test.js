// @flow

import React from 'react';
import { shallow } from 'enzyme';
import TodoList from './TodoList';

describe('TodoList component', () => {
  it('renders as expected', () => {
    const component = shallow(
      <TodoList>
        <p>a child</p>
      </TodoList>,
    );
    expect(component).toMatchSnapshot();
  });
});
