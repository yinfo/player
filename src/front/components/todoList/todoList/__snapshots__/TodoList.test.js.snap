// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoList component renders as expected 1`] = `ShallowWrapper {}`;
